tasks/extract_filepaths_for_granule/extract_filepaths_for_granule.py

```diff
@@ -52,13 +66,18 @@ def task(event, context):    #pylint: disable-msg=unused-argument
                 level = "event['input']['granules'][]['files']"
```

Comment:
I would move this into the exception block wince it is only used there.

Reply:
I am getting some errors on unit test when I moved that ```level``` under exception block therefore keeping it as it is for now.

Reply:
What is the unit test error?

Reply:
@bhazuka 
```
============================================================================ short test summary info ============================================================================
FAILED test/unit_tests/test_extract_file_paths_for_granule.py::TestExtractFilePaths::test_task_no_files - AssertionError: 'KeyError: "event[\'input\'][\'granules\'][][\'files...
FAILED test/unit_tests/test_extract_file_paths_for_granule.py::TestExtractFilePaths::test_task_no_granule - AssertionError: 'KeyE[16 chars]nput\'][\'granules\'][][\'granuleId...
FAILED test/unit_tests/test_extract_file_paths_for_granule.py::TestExtractFilePaths::test_task_no_granules - AssertionError: 'KeyE[15 chars]input\'][\'granules\']" is required...
```


```
>           self.assertEqual(exp_err, str(ex))
E           AssertionError: 'KeyE[15 chars]input\'][\'granules\']" is required' != 'KeyE[15 chars]input\'][\'granules\'][][\'files\'][\'granules\']" is required'
E           - KeyError: "event['input']['granules']" is required
E           + KeyError: "event['input']['granules'][]['files']['granules']" is required
E           ?                                      +++++++++++++++++++++++
```